core/templates/dev/head/library/library.html

```diff
@@ -56,16 +56,16 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
                translate="I18N_LIBRARY_NO_EXPLORATION_GROUPS">
           </div>
 
-          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index">
+          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index" ng-init="groupIndex = $index">
```

Comment:
Yes. But I'm calling the function highlightGroup inside tiles (ng-mouseover). So I need the groupIndex to be supplied as the argument. But as it turns out $index here refers to the tile index.